README.md

```diff
@@ -29,7 +29,7 @@ Check individual folders for usage guidelines and examples.
 ## Supported Rust Versions
 
 OpenTelemetry is built against the latest stable release. The minimum supported
-version is 1.75. The current OpenTelemetry version is not guaranteed to build
+version is 1.85. The current OpenTelemetry version is not guaranteed to build
```

Comment:
Don't update the compiler requirements for all the packages within this repo. Only do it for `opentelemetry-geneva-exporter/*/`